### ISSUE TITLE
Post deploy script

### ISSUE THREAD

**Issue by daftspunk** [rainlab/deploy-plugin#38]
Ability to run artisan commands every time a deployment happens, eg `npm:production`

**Comment by adrian-stein** [comment]
Yes, this would be super useful and truly make it a one touch deploy process.
